src/AppBundle/Entity/User.php

```diff
@@ -12,7 +12,7 @@
  * Users.
  *
  * @ORM\Table(name="dd_user", indexes={@ORM\Index(name="deputy_no_idx", columns={"deputy_no"})})
- * @ORM\Entity(repositoryClass="AppBundle\Entity\Repository\UserRepository")
```

Comment:
why is this removed? (curiosity)

Reply:
One of the benefits of making the Repository classes standalone, without extending Doctrine, is that we can have many Repo's for one Entity, and therefore don't need to declare this here.